### PR TITLE
Output options in unicode and needed to be cast to strings

### PR DESCRIPTION
**Pull request by oakeyc**

### CODE REVIEW

File: azclishell/util.py [hunk 1]
```diff
@@ -119,4 +119,8 @@ def parse_quotes(cmd, quotes=True):
             args.extend(words.split())
     else:
         args = words.split()
-    return args
+
+    str_args = []
+    for arg in args:
+        str_args.append(str(arg))
```
Comment (derekbekoe) [inline azclishell/util.py:125]: Does this work well for both Python 2 and 3?

File: setup.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 DEPENDENCIES = [
     'applicationinsights',
-    'azure-cli',
+    'azure-cli==2.0.2',
```
Comment (derekbekoe) [inline setup.py]: Shouldn't pin to a specific CLI version as this doesn't work as intended.
You'll get all the latest modules still.

Comment (oakeyc) [reply setup.py]: ok

File: azclishell/util.py (hunk 1 above)
Comment (oakeyc) [reply azclishell/util.py:125]: Yes

**Review by derekbekoe — APPROVED** [review]

**Pull request merged**